src/less/main.less

```diff
@@ -571,6 +571,12 @@
 .braintree-sheet__content--form {
   padding: 10px 15px 10px 10px;
 
+  .braintree-form__notice-of-collection {
+    font-size: 14px;
```

Comment:
Just my two cents, but I would make this font size smaller. I was thinking similar to the `Having trouble logging in?` text size relative to the fields in the below example:

![netflix](https://github.com/braintree/braintree-web-drop-in/assets/35243507/3616587f-e477-4fda-b81d-c07983b3fd77)

Reply:
Reduced to 12px, updated the screenshots in the description.